modules/core/src/matrix.cpp

```diff
@@ -4277,6 +4277,17 @@ Rect RotatedRect::boundingRect() const
     return r;
 }
 
+Rect_<float> RotatedRect::boundingRect2f() const
+{
+    Point2f pt[4];
+    points(pt);
+    Rect_<float> r(min(min(min(pt[0].x, pt[1].x), pt[2].x), pt[3].x),
+                   min(min(min(pt[0].y, pt[1].y), pt[2].y), pt[3].y),
+                   max(max(max(pt[0].x, pt[1].x), pt[2].x), pt[3].x),
+                   max(max(max(pt[0].y, pt[1].y), pt[2].y), pt[3].y));
```

Comment:
There is used wrong `Rect_` constructor:
`Rect_(_Tp _x, _Tp _y, _Tp _width, _Tp _height)`

Probably you need to use this:
`Rect_(const Point_<_Tp>& pt1, const Point_<_Tp>& pt2)`